sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseService.java

```diff
@@ -714,6 +711,19 @@ public Predicate createChangeDateFilter(CriteriaBuilder cb, From<?, Case> casePa
 		dateFilter = cb.or(dateFilter,
 				greaterThanAndNotNull(cb, portHealthInfo.get(AbstractDomainObject.CHANGE_DATE), date));
 
+		if (includeAdditionalParams) {
+			Join<Case, Sample> caseSampleJoin = casePath.join(Case.SAMPLES, JoinType.LEFT);
+			dateFilter = cb.or(dateFilter,
+					greaterThanAndNotNull(cb, caseSampleJoin.get(AbstractDomainObject.CHANGE_DATE), date));
+
+			Join<Sample, PathogenTest> samplePathogenTestJoin = caseSampleJoin.join(Sample.PATHOGENTESTS, JoinType.LEFT);
+			dateFilter = cb.or(dateFilter,
+					greaterThanAndNotNull(cb, samplePathogenTestJoin.get(AbstractDomainObject.CHANGE_DATE), date));
+
+			Join<Case, Person> casePersonJoin = casePath.join(Case.PERSON, JoinType.LEFT);
+			dateFilter = cb.or(dateFilter, greaterThanAndNotNull(cb, casePersonJoin.get(AbstractDomainObject.CHANGE_DATE), date));
```

Comment:
You also need to check the person address change date

Reply:
updated